test/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -770,20 +770,23 @@ end
     @test !MOIB.is_variable_bridged(bridged, cy)
     @test MOIB.bridge(bridged, cy) isa MOIB.Constraint.RSOCBridge{T}
     @test sprint(MOIB.print_graph, bridged) == """
-Bridge graph with 4 variable nodes, 9 constraint nodes and 0 objective nodes.
+Bridge graph with 5 variable nodes, 11 constraint nodes and 0 objective nodes.
```

Comment:
@blegat These are changes that were outside of the scope of my additions. I figured it's ok because this is print output for debugging I gues.

Reply:
Yes, it is an expected side effect